lua/neogit/buffers/common.lua

```diff
@@ -88,6 +88,25 @@ local function build_graph(graph)
   end
 end
 
+-- - '%G?': show "G" for a good (valid) signature,
+--   "B" for a bad signature,
+--   "U" for a good signature with unknown validity,
+--   "X" for a good signature that has expired,
+--   "Y" for a good signature made by an expired key,
+--   "R" for a good signature made by a revoked key,
+--   "E" if the signature cannot be checked (e.g. missing key)
+--   and "N" for no signature
+local highlight_for_signature = {
+  G = "String", -- Green
+  B = "Error", -- Red
+  U = "Operator", -- Cyan
+  X = "SpecialChar", -- Orange
+  Y = "SpecialChar", -- Orange
+  R = "Error", -- Red
+  E = "Special", -- Blue
+  N = "Comment", -- Default
+}
```

Comment:
Check out `hl.lua` - we have highlight groups for each color. They're only used for the `--color` flag when viewing the log graph, but no reason they can't be used here too.